### PR TITLE
atdm/contributed: environment script for blake testbed

### PR DESCRIPTION
**Pull request by ndellingwood**
## Motivation
Add testing script for Blake testbed

## Testing

Successfully tested build with Kokkos and KokkosKernels enabled with two configurations:

```bash
export ATDM_CONFIG_REGISTER_CUSTOM_CONFIG_DIR=${TRILINOS_DIR}/cmake/std/atdm/contributed/blake
# Configure 1
source ${TRILINOS_DIR}/cmake/std/atdm/load-env.sh gnu-serial
# Configure 2
source ${TRILINOS_DIR}/cmake/std/atdm/load-env.sh intel-serial
```

### CODE REVIEW

File: cmake/std/atdm/contributed/blake/environment.sh [hunk 1]
```diff
@@ -0,0 +1,212 @@
+################################################################################
+#
+# Set up env on a SEMS NFS mounted RHEL7 for ATMD builds of Trilinos
```
Comment (bartlettroscoe) [inline cmake/std/atdm/contributed/blake/environment.sh:3]: Might want to fix the comment to say 'blake' or just remove this.

File: cmake/std/atdm/contributed/blake/environment.sh [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+# file that it passes to 'nvcc' to create the kokkoskernels library.  When the
+# CMake-generated *.rsp file is passed to 'nvcc', it does not know how to
+# process it and it gives the error "No input files specified".  The
+# workaround is to switch to the CMake Makefile generator which allows and
+# turning off the usage of response files for the list of object files (and
+# 'nvcc' seems to be able to handle this long command-line in this case).
+# Note that we don't yet need to switch to use the CMake Makefile generator
+# for 'static' builds since 'ar' is used to create the kokkoskernels lib which
+# does not seem to have a problem with the creation of this lib.  (See
+# TRIL-255 and TRIL-264.)
```
Comment (bartlettroscoe) [inline cmake/std/atdm/contributed/blake/environment.sh:173]: Is this still true?  Just asking.

**Review by bartlettroscoe — APPROVED** [review]
@ndellingwood, if it works for you, then that is good enough for me.

Just made a few comments/suggestions.  Feel free to ignore.

File: cmake/std/atdm/contributed/blake/environment.sh (hunk 2 above)
Comment (ndellingwood) [reply cmake/std/atdm/contributed/blake/environment.sh:173]: Oop, this is copy-paste, I'll delete; I'm not sure about the current status of that note on the relevant machines.

Comment (ndellingwood) [reply cmake/std/atdm/contributed/blake/environment.sh]: @bartlettroscoe I deleted the comment (and logic above it), but my commit dismissed your review, can you approve one more time please?

**Review by bartlettroscoe — APPROVED** [review]

**Pull request merged**